src/tsserverlibrary/_namespaces/ts.server.ts

```diff
@@ -1,4 +1,5 @@
 /* Generated file to emulate the ts.server namespace. */
 
 export * from "../../jsTyping/_namespaces/ts.server";
+export * from "../../typingsInstallerCore/_namespaces/ts.server";

```

Comment:
If you have already done this from `ts.server.ts` in `src/server`, you shoudn't need to do that here (or modify tsconfig.json).